packages/design-system/src/components/button.tsx

```diff
@@ -49,17 +51,25 @@ const foregrounds: Record<ButtonColor, string> = {
   ghost: theme.colors.foregroundMain,
   dark: theme.colors.foregroundContrastMain,
   gradient: theme.colors.foregroundContrastMain,
+  "dark-ghost": theme.colors.foregroundContrastMain,
```

Comment:
hmm did we use dash before here? or camel case?

Reply:
in figma here is space, but variants are not working with space. For -neutral style above we use dash

Reply:
what have we done so far? did we convert space to something else? or is it first time figma uses space and should be fixed in figma?

Reply:
@taylornowotny was "dark ghost" first variant that uses space?

Reply:
ok i'll add a dash. For figma it doesn't matter.

Reply:
<img width="160" alt="Screenshot 2023-10-07 at 9 58 42 AM" src="https://github.com/webstudio-is/webstudio/assets/86495896/31d924fb-1588-4a9c-9bcc-1820b194493c">
it is done